make/common/NativeCompilation.gmk

```diff
@@ -1163,7 +1163,7 @@ define SetupNativeCompilationBody
                 # This only works if the openjdk_codesign identity is present on the system. Let
                 # silently fail otherwise.
                 ifneq ($(CODESIGN), )
-		  $(CODESIGN) -s "$(MACOSX_CODESIGN_IDENTITY)" --timestamp --options runtime \
+		  $(CODESIGN) -f -s "$(MACOSX_CODESIGN_IDENTITY)" --timestamp --options runtime \
```

Comment:
I ran into the same issue, interestingly only when doing cross-compilation from x86_64.  `strip` is suppose to remove the signature, but `codesign -s` claims a signature is still there. However if you inspect the binary with `codesign -d -r-` the signature is indeed gone. Doing the change shown as here helped and I couldn't observe any issues with it. Did you experience the same thing?

Reply:
well, no, strip was fixed to keep adhoc signature some time ago

Reply:
I missed that, thank you.